packages/data-broker/src/getModels.js

```diff
@@ -1,14 +0,0 @@
-/**
- * Tupaia
- * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
- */
-
-import { ModelRegistry, TupaiaDatabase } from '@tupaia/database';
-
-import { modelClasses } from './modelClasses';
-
-const database = new TupaiaDatabase();
```

Comment:
So this is what was causing the issue...  The database will be unconditionally instantiated every times this file is loaded